livekit-agents/livekit/agents/voice/agent_activity.py

```diff
@@ -2654,7 +2675,9 @@ def _on_false_interruption() -> None:
                 and not self._paused_speech.done()
             ):
                 self._session._update_agent_state(
-                    "speaking", otel_context=self._paused_speech._agent_turn_context
+                    "speaking",
+                    otel_context=self._paused_speech._agent_turn_context,
+                    start_time=time.time_ns(),
```

Comment:
```suggestion
                    start_time_ns=time.time_ns(),
```
to make it more explicit?